### ISSUE TITLE
PyCharm plugin build fails

### ISSUE THREAD

**Issue by daghoidahl** [reloadware/reloadium#18]
I'm trying to build the plugin myself from the `main` branch on a MacBook Pro M1, but it fails.

First I need to remove some misplaced changelog text from `pycharm/src/main/java/rw/profile/ProfileResultToggle.java`. 

After that, I get this message:

```
❯ gradle buildPlugin

> Task :compileJava FAILED
/Users/dagh/programs/reloadium/pycharm/src/main/java/rw/handler/sdk/BaseSdkHandler.java:6: error: cannot find symbol
import rw.consts.Const;
                ^
  symbol:   class Const
  location: package rw.consts

  [same error is repeated for a bunch of locations]
```

_Originally posted by @daghoidahl in https://github.com/reloadware/reloadium/issues/3#issuecomment-1144893841_

**Comment by dkrystki** [comment]
Fixed in PyCharmPlugin [0.8.3]

**Comment by daghoidahl** [comment]
> Fixed in PyCharmPlugin [0.8.3]

Brilliant! This problem has been solved.

I have a new problem now that may or may not be related to the plugin build scripts:

```
Missing essential plugin:

  com.jetbrains.pycharm.community.customization

Please reinstall PyCharm from scratch.
```

I have PyCharm professional, could that be the cause of the issue?
